src/scriptdev2/scripts/outland/tempest_keep/the_eye/boss_alar.cpp

```diff
@@ -156,17 +157,64 @@ struct boss_alarAI : public ScriptedAI
                 pSummoned->AI()->AttackStart(pTarget);
         }
     }
-
+    
     void SummonedCreatureJustDied(Creature* pSummoned) override
     {
-        // drain 3% of boss health when the ember dies
+        // drain 2% of boss health when the ember dies
```

Comment:
You changed the comment, but not the value in the code bellow.

According to wow wiki it's indeed 2% and not 3.